rakelib/test.rake

```diff
@@ -24,7 +24,7 @@ namespace "test" do
     # logstash-core-event specs since currently this is the most complete Event and Timestamp specs
     # which actually defines the Event contract and should pass regardless of the actuall underlying
     # implementation.
-    specs = ["spec/**/*_spec.rb", "logstash-core/spec/**/*_spec.rb", "logstash-core-event/spec/**/*_spec.rb"]
+    specs = ["test/unit/**/*_spec.rb"]
```

Comment:
@andrewvc after thinking a bit more on this I have a few questions about it, I would really love to understand better:

- logstash-core-event test include event_spec but also timestamp_spec, the first one I understand belongs to the event, but not the timestamp. Do you know about the relation of both?

- I understand we want to extract the java-event, but is still having a logstash-event "gem" something meaningful to us? does it makes sense? what are the benefits we get for it?